lib/pagerduty/pagerduty.rb

```diff
@@ -535,6 +535,37 @@ def notes(id)
     }))
   end
 
+  # Create an note for a specific incident.
+  #
+  # ==== Parameters
+  # * 'options'<~Hash>
+  #   * 'id' - the id of the incident
+  #   * 'requester_id'<~String>
+  #   * 'note' - <~Hash>
+  #      * 'content' - <~ String>
+  # ==== Returns
+  # * <~Pagerduty::Schedules::Overrides::Override>
+  #   * 'id'<~String>
+  #   * 'user'<~Pagerduty::User>
+  #     * 'id'<~String>
+  #     * 'name'<~String>
+  #     * 'email'<~String>
+  #     * 'color'<~String>
+  #     * 'role'<~String>
+  #     * 'avatar_url'<~String>
+  #     * 'user_url'<~String>
+  #     * 'invitation_sent'<~Boolean>
+  #     * 'marketing_opt_out'<~Boolean>
+  #   * 'content'<~String>
+  #   * 'created_at'<~String>
+  # {Pagerduty API Reference}[https://developer.pagerduty.com/documentation/rest/schedules/overrides/create]
+  def create_note(options={})
+    Pagerduty::Schedules::Overrides::Override.new(curl({
```

Comment:
Why are you returning this object type? The response from the API should be a Note object. (See: https://developer.pagerduty.com/documentation/rest/incidents/notes/create)